cura/CuraApplication.py

```diff
@@ -1457,7 +1458,11 @@ def updatePlatformActivity(self, node = None):
             self._scene_bounding_box = scene_bounding_box
             self.sceneBoundingBoxChanged.emit()
 
-        self._platform_activity = True if count > 0 else False
+        if count > 0:
+            ProjectOutputDevice.setLastOutputName(None)
```

Comment:
I'm not familiar with this part of Cura but I wonder if the call to setLastOutputName(None) should be done when count is 0 rather than when count > 0 ?

Reply:
I think you're right -- I'm not _entirely_ sure why it also seems to work when the current code is run then.

Reply:
Fortunately, the fix still works the other way around. I changed it to the code that makes more sense.